admin/package.json

```diff
@@ -47,7 +47,7 @@
     "dev": "./node_modules/.bin/webpack-dev-server --mode development",
     "build": "./node_modules/.bin/webpack",
     "test": "jest",
-    "eslint": "eslint src"
+    "eslint": "eslint srr --ext .js --ext .jsx"
```

Comment:
This looks like a typo. Shouldn't it still be 'src' and not 'srr'?

Reply:
This is a typo alright

Reply:
Should be fixed now! Also eslint issues should have been addressed.